pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -523,6 +544,11 @@ public void destroy() {
       segmentLogger.error("Could not stop consumer thread");
     }
     _realtimeSegment.destroy();
+    try {
+      _consumerWrapper.close();
```

Comment:
Not 100% sure, but I think you can do `IOUtils.closeQuietly(_consumerWrapper)` so that you don't have to catch the exception